### ISSUE TITLE
Disabled button style

### ISSUE THREAD

**Issue by panr** [18F/web-design-standards#944]
**Description:**

Disabled button has vivid font color. Even when background of button is light gray and cursor is set to default, it still looks clickable (in my opinion).

**Ask:**

It would be great if disabled button could have less vivid font color. Maybe `rgba(0, 0, 0, 0.3);`?

**To Do:**
- [ ] Set less vivid font color in disabled button (`.usa-button-disabled` and reset `.usa-button-disabled:hover` to its default value)

**How I see this after change:**

<img width="419" alt="disabled_button" src="https://cloud.githubusercontent.com/assets/1303365/12093943/22d426f6-b306-11e5-9fc8-edaa763b1c2c.png">

**Comment by maya** [comment]
That's a good point, @panr. Your modification definitely looks more disabled than our button. Cc-ing @nickbristow to review for accessibility reasons.

Thanks for the suggestion!

**Comment by maya** [comment]
Looks like there are no color contrast requirements for disabled states. See - 1.4.3: http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

>Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

**Comment by panr** [comment]
So we can change background color of disabled button to even lighter gray (I think) :-)
And of course every disabled element should use the same color pallete.

**Comment by bbertucc** [comment]
Any updates to the disabled button should also anticipate disabled outline buttons. I added some examples, inspired your suggestions @panr, to issue #947

**Comment by nickbristow** [comment]
@maya this is correct, no color contrast requirement under WCAG 2.0 AA, 

Note though, there is a contrast requirement for DHS for disabled buttons, but we are following WCAG for the standards, so go for it.

**Comment by humancompanion** [comment]
@maya - We also felt the disabled button didn't look quite disabled enough. In addition, we had an issue where a primary button was placed on a grey background, which worked fine until the button was disabled and then it matched the background color. In that case we made a special exception and altered the opacity. That combined with another instance (in our Document component) lead me to alter the opacity everywhere:

http://usajobs.github.io/design-system/buttons/

Here is the special instance:
<img width="1102" alt="screen shot 2016-01-11 at 8 52 40 am" src="https://cloud.githubusercontent.com/assets/352987/12237445/b2dc2bd6-b84c-11e5-911a-70fec4a7a1ec.png">

I noticed the trend is towards making disabled buttons gray (salesforce, material design), but what I've seen done in the past is retaining the original style of the button, not altering the classes but just adding the disabled attribute, and using opacity to style the disabled state. Was that considered and discarded? It would take more styling to handle hover and other states, but it would retain the intent of the button (primary still looks like primary, secondary still looks like secondary). For example, the instance above where a primary button is disabled:

<img width="1102" alt="screen shot 2016-01-11 at 8 53 35 am" src="https://cloud.githubusercontent.com/assets/352987/12237515/14c7d9bc-b84d-11e5-8424-242fdbc1e1a8.png">

**Comment by maya** [comment]
We have a new disabled button that follows this recommendation: https://standards.usa.gov/components/buttons/

/cc @thisisdano @donjo just wanted you to be aware of the different background color issue and the opacity approach @humancompanion took.
